### PR TITLE
Replaced Utils.keys with Object.keys, Utils.arrayIndexOf with Array.indexOf

### PR DESCRIPTION
**Pull request by allmywant**
And some changes for ESLint warnings.

### CODE REVIEW

File: lib/mojito.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ Mojito = (function () {
                 if (previewRecipe != null) {
                     this.log('Forcing test [' + this.options.name + '][' + this.options.id + '] into recipe [' + previewRecipe.name + '][' + recipeId + ']');
 
-                    this.setInTest(1);
+                    this.setInTest('1');
```
Comment (kingo55) [inline lib/mojito.js:111]: Were there type problems related to Utils.keys/arrayIndexOf, that this addresses?

**Review by kingo55 — APPROVED** [review]
Looks good to me!

File: lib/mojito.js (hunk 1 above)
Comment (allmywant) [reply lib/mojito.js:111]: Yes, but this is mainly for the changes: `==` to `===`.

Comment (kingo55) [reply lib/mojito.js:111]: Cool - thought so.

**Review by dapperdrop — APPROVED** [review]

**Pull request merged**